src/domain/interface.ts

```diff
@@ -260,7 +260,6 @@ export interface Sections {
 export interface Application extends ApplicationBaseFields {
   appNumber: number;
   submitterEmail: string;
-  submittedAtUtc: Date;
```

Comment:
why is this deleted in a commit that only mentions swagger updates?

Reply:
because i failed you @joneubank 
pr description is now updated